src/integrationTest/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersControllerIT.java

```diff
@@ -124,6 +126,110 @@ void setUp() {
 
         }
 
+
+        @Nested
+        @DisplayName("Negative Authentication Tests (GET)")
+        class NegativeAuthTest implements BaseMvcAuthChecks {
+
+            private MockHttpServletRequestBuilder happyPathRequestBuilder;
+
+            @BeforeEach
+            public void setUp() throws JsonProcessingException {
+                // NB: need a unique case ID when running happy path for a second time.
+                caseId = uidService.generateUID();
+                createCasePaymentOrderRequest = new CreateCasePaymentOrderRequest(caseId,
+                                                                                  ACTION, RESPONSIBLE_PARTY,
+                                                                                  ORDER_REFERENCE_VALID
+                );
+
+                happyPathRequestBuilder = post(CASE_PAYMENT_ORDERS_PATH)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(objectMapper.writeValueAsString(createCasePaymentOrderRequest));
+            }
+
+            @Test
+            @Override
+            @DisplayName(DISPLAY_ALL_AUTH_OK)
+            public void should2xxSuccessfulForHappyPath() throws Exception {
```

Comment:
Can we extract ALL security tests into another class file in which **service** is mocked ? Also all negative IT tests don't even bother about payload structure. So any chance to simplify all these tests with @ParameterizedTest and @MethodSource ?

Reply:
Ignore for now if it's a big change or doesn't add much value.